CloudWatch2S3.template

```diff
@@ -663,7 +670,16 @@ Resources:
 
             def subscribe_all():
                 for log_group_name in matched_log_groups("${LogGroupNamePrefix}"):
-                    subscribe(log_group_name)
+                    if not "${LogGroupTagName}":
+                        subscribe(log_group_name)
+                    else: 
+                        tags_in_group_response = logs_client.list_tags_log_group(
+                            logGroupName = log_group_name
+                        )
+                        print("tags_in_group_response", tags_in_group_response)
+                        for tag in tags_in_group_response["tags"]:
+                            if tag == "${LogGroupTagName}":
+                                subscribe(log_group_name)
```

Comment:
Can we add a `break` here just in case? Don't want for it to be possible to get double subscribe (which I think results in an exception).